README.rdoc

```diff
@@ -44,7 +44,11 @@ The plugin supports the following parameters:
 
 [message]  Name of the field in the JSON record that contains the
            single-line log messages that shall be scanned for exceptions.
-           Default: 'message'.
+           If the setting is '', it will automatically be changed to 'message'
```

Comment:
`This parameter is only applicable to structured (JSON) log streams.` Also below.